css-three-column-layout/styles.css

```diff
@@ -0,0 +1,54 @@
+.container {
+  max-width: 1000px;
+  min-height: 350px;
+  margin: 0 auto;
+  background-color: #1297db;
```

Comment:
Apply the `background-color` to the `body` rather than the `container` so that it fills up the whole page